spec/mid_mod_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe 'as a user' do
+  it 'I can search foods' do
+    response = File.read("#{Rails.root}/spec/fixtures/food_api/sweet_potato_10_results_search.json")
+    parsed_response = JSON.parse(response, symbolize_names: true)[:foods].first
+
+    stub_request(:get, "https://api.nal.usda.gov/fdc/v1/foods/search?api_key=#{ENV['FOOD_API_KEY']}&pageSize=10&query=sweet%20potatoes").to_return(status: 200, body: response, headers: {})
+
+    visit root_path
+
+    fill_in :q, with: "sweet potatoes"
+    click_button "Search"
+    expect(current_path).to eq("/foods")
+
+    # and see the total numbers of search results
+    expect(page).to have_content("Total Results: 45635")
+
+    #and see 10 foods containing my search term
+    expect(page).to have_css('.single-result', count: 10)
+
+    #for each food I should see details
+    # within(".single_result", match: :first) do
```

Comment:
to target a specific part of the page, you may consider having something dynamic and unique that you could call this class. Maybe you also add the id number for that food onto the end of that class name?  Then you could target the specific area of the page where you'd expect that data to render.

Reply:
ended up commenting that out because I was having trouble figuring out what I was looking for on the page. I originally went with a class, because I imagine I would want each result identically styled...I think I need more hands on practice with html/css styling choices